### PR TITLE
Freightliner cascadia

### PR DESCRIPTION
**Pull request by as-snehagn**
Adding new vehicle type - Freightliner Cascadia.

### CODE REVIEW

File: launch/pacmod_game_control.launch [hunk 1]
```diff
@@ -22,10 +22,11 @@
        POLARIS_RANGER
        LEXUS_RX_450H
        INTERNATIONAL_PROSTAR_122
+       FREIGHTLINER_CASCADIA
        VEHICLE_4
        VEHICLE_5
        VEHICLE_6 -->
-  <arg name="pacmod_vehicle_type" default="LEXUS_RX_450H" />
+  <arg name="pacmod_vehicle_type" default="POLARIS_GEM" />
```
Comment (JWhitleyAStuff) [inline launch/pacmod_game_control.launch]: Please change this back to LEXUS_RX_450H. Changing the default launch parameters is a bad idea. Customers rely on them.

File: src/publish_control_board_rev3.cpp [hunk 2]
```diff
@@ -67,6 +67,8 @@ void PublishControlBoardRev3::publish_steering_message(const sensor_msgs::Joy::C
   float range_scale;
   if (vehicle_type == VEHICLE_4 || vehicle_type == VEHICLE_6 || vehicle_type == LEXUS_RX_450H)
     range_scale = 1.0;
+  else if (vehicle_type == FREIGHTLINER_CASCADIA)
+    range_scale = 1.0;  // testing
```
Comment (JWhitleyAStuff) [inline src/publish_control_board_rev3.cpp]: This can be combined with the `if` above this. Also, please remove comment.

Comment (as-snehagn) [reply src/publish_control_board_rev3.cpp]: I wanted to test it on the truck before combining it in the if.

**Comment by JWhitleyAStuff** [comment]
PR is holding for testing per @as-snehagn.

**Comment by JWhitleyAStuff** [comment]
@as-snehagn - Please rebase your branch on `master` to fix the conflicts.

**Comment by JWhitleyAStuff** [comment]
@as-snehagn - All done with testing? All good?

**Comment by as-snehagn** [comment]
@JWhitleyAStuff Testing looks good for the existing buttons. Do we want to add some of the additional features as buttons on the game pad?

**Comment by JWhitleyAStuff** [comment]
@as-snehagn - I would say no. We have already used up all of the available buttons and are starting to get into multiple-button combos for special use cases. Better to leave the controller app pretty basic, IMO so it works basically the same on all vehicles.

**Pull request merged**